src/config/app.ts

```diff
@@ -0,0 +1,13 @@
+import dotenv from 'dotenv';
+import path from 'path';
+
+// Make environment variables available in application
+dotenv.config({ path: path.resolve(__dirname, '../../.env') });
+
+export default {
+  port: process.env.PORT || '3000',
+  clientId: process.env.CLIENT_ID || '',
+  clientSecret: process.env.SECRET || '',
+  redirectUrl: process.env.REDIRECT_URL || '',
+  authorizationHost: process.env.AUTHORIZATION_HOST || '',
+};
```

Comment:
src/config/app.ts seems to add a layer of complexity that does not seem to be necessary.  Yes, it matches some of the examples we've seen but I don't see that it really adds anything.  The `dotenv` component already encapsulates the configuration values.

At this point, please do not remove it if it hasn't broken anything.

Reply:
It hasn't broken anything, so I think we're not gonna have any problems implementing like this.